### ISSUE TITLE
Tickets in feedMe?

### ISSUE THREAD

**Issue by anchovy** [verbb/Events#145]
### Question

Can we import tickets via feedme? it's mentioned in #46  but this is quite old.
The ticket options are visible when setting up the feedme feed, but neither the ticket settings capacity field is added when run.
![image](https://github.com/verbb/events/assets/599005/ad2e079c-e7ee-46f0-9305-9b22c797596d)

**Comment by engram-design** [comment]
Seems to work for me using JSON data.

```json
[
    {
        "title": "Feed Me Test",
        "tickets": [
            {
                "type": 158,
                "price": 11,
                "quantity": 9999
            }
        ]
    }
]
```

Note that that `type` is the Ticket Type ID. I can see you're just using default values, but the integration doesn't work like that and needs to be provided with an array of ticket data. It's possible we can make allowances for this scenario if that's valid - I just wasn't sure if you were doing that just for testing.

**Comment by anchovy** [comment]
Hi, thanks for that. In our use case we do need default tickets for all imported events, so using default values is a valid use case.

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, run `composer require verbb/events:"dev-craft-4 as 2.0.5"`.

**Comment by engram-design** [comment]
Fixed in [2.0.6](https://github.com/verbb/events/releases/tag/2.0.6)
